emission/individual_tests/TestNominatim.py

```diff
@@ -0,0 +1,113 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import *
+import unittest
+import unittest.mock as mock
+# import overpy
+import os
+from emission.core.wrapper.trip_old import Coordinate
+import requests
+import emission.core.wrapper.entry as ecwe
+import emission.core.wrapper.wrapperbase as ecww
+import emission.net.ext_service.geocoder.nominatim as eco
+import emission.analysis.intake.cleaning.clean_and_resample as clean
+
+#temporarily sets NOMINATIM_QUERY_URL to the environment variable for testing.
+NOMINATIM_QUERY_URL_env = os.environ.get("NOMINATIM_QUERY_URL")
```

Comment:
"Temporary" in the sense that in production it will be the geofabrik url, and in the test it will be the container name! So "temporarily" changing from the default for the purpose of the test.

Reply:
Changed :)

Reply:
please see review:
https://github.com/e-mission/e-mission-server/pull/929#pullrequestreview-1651639356